app/src/main/java/in/org/celesta/iitp/sponsors/SponsorsFragment.java

```diff
@@ -48,15 +50,45 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
+        sponsorItems.add(new SponsorItem("Food Partner", "http://www.celesta.org.in/assets/images/sponsors/chicken.jpeg", "https://broaster.com/"));
+        sponsorItems.add(new SponsorItem("Training Partner", "http://www.celesta.org.in/assets/images/sponsors/certstore.png", "https://in.linkedin.com/company/cert-store-solution"));
+        sponsorItems.add(new SponsorItem("Implementation Partner", "http://www.celesta.org.in/assets/images/sponsors/youthindiafoundation.png", "https://youngindia.foundation/"));
+        sponsorItems.add(new SponsorItem("Implementation Partner", "http://www.celesta.org.in/assets/images/sponsors/techprolabz.jpeg", "http://www.techprolabz.com/"));
+        sponsorItems.add(new SponsorItem("Web&SMS Partner", "http://www.celesta.org.in/assets/images/sponsors/asap.png", "https://www.asapservices.co.in/services.html"));
+        sponsorItems.add(new SponsorItem("Merchandise Partner", "http://www.celesta.org.in/assets/images/sponsors/awestruck.jpeg","https://www.getawestruck.com/" ));
+        sponsorItems.add(new SponsorItem("Workshop Partner", "http://www.celesta.org.in/assets/images/sponsors/whataftercollege.png", "https://whataftercollege.com/"));
+        sponsorItems.add(new SponsorItem("Blogger Outreach Partner", "http://www.celesta.org.in/assets/images/sponsors/blogadda2.png", "https://www.blogadda.com/"));
+        sponsorItems.add(new SponsorItem("Magazine Partner", "http://www.celesta.org.in/assets/images/sponsors/scientificindia.png", "http://www.scind.org/"));
+        sponsorItems.add(new SponsorItem("Online Media Partner", "http://www.celesta.org.in/assets/images/sponsors/festpav.png", "http://festpav.com/"));
+        sponsorItems.add(new SponsorItem("Online Media Partner", "http://www.celesta.org.in/assets/images/sponsors/ntdindia.jpg", "http://ntdin.tv/"));
+        sponsorItems.add(new SponsorItem("Online Media Partner", "http://www.celesta.org.in/assets/images/sponsors/The-Bihar-News.png", "https://www.thebiharnews.in/"));
+        sponsorItems.add(new SponsorItem("Online Media Partner", "http://www.celesta.org.in/assets/images/sponsors/thecollegefever.png", "https://www.thecollegefever.com/"));
+        sponsorItems.add(new SponsorItem("Voice Opinion Partner", "http://www.celesta.org.in/assets/images/sponsors/opentalk.png", "https://opentalk.to/"));
+        sponsorItems.add(new SponsorItem("Gifting Partner", "http://www.celesta.org.in/assets/images/sponsors/thesouledstore.png", "https://www.thesouledstore.com/"));
+        sponsorItems.add(new SponsorItem("Online Learning Partner", "http://www.celesta.org.in/assets/images/sponsors/learncodeonline.png","https://learncodeonline.in/" ));
+        sponsorItems.add(new SponsorItem("Official Savings Partner", "http://www.celesta.org.in/assets/images/sponsors/grabon.png", "https://www.grabon.in/"));
+        sponsorItems.add(new SponsorItem());
```

Comment:
This space left due to vacancy at Printing Partner section in the official webpage(of celesta2k19)